### ISSUE TITLE
Tango eyes are blurred

### ISSUE THREAD

**Issue by archisman-panigrahi** [luisbocanegra/plasma-cursor-eyes#4]
The eyes have somewhat low resolution in the Tango theme. Other themes look great!

![image](https://github.com/user-attachments/assets/2e126cf7-0a95-4832-b56a-82bb0405dac1)

**Comment by archisman-panigrahi** [comment]
The temporary workaround is to have smaller eyes with smaller iris and pupil (absolute size).

**Comment by luisbocanegra** [comment]
Yeah, could use some designer redoing some of the images, even recreating the simpler ones as svg files could be an option.
